content/product-engineering/engineering/enablement/frontend-platform/other-responsibilities.md

```diff
@@ -5,16 +5,16 @@ team](./index.md).
 
 ## @frontend-devs
 
-The [Frontend Platform team](./index.md) maintains the canonical list of frontend developers at Sourcegraph, which includes all engineers with expertise in HTML, CSS, and JavaScript/TypeScript (even if they don't exclusively do frontend development).
-
-The people on this list may be called on to:
+The [Frontend Platform team](./index.md) maintains the canonical list of frontend developers at Sourcegraph. These engineers all have expertise in HTML, CSS, JavaScript/TypeScript, and React, and maybe be called on to:
 
 - give their opinions on frontend tooling at Sourcegraph
 - review frontend PRs
 - do frontend interviews and assignment grading
 
 These people are available under the `@frontend-devs` alias on both Slack and [GitHub](https://github.com/orgs/sourcegraph/teams/frontend-devs) (the two lists are kept in sync). Please contact the Frontend Platform team to add or remove someone from the list.
 
+Note that this list is _not_ intended to be a comprehensive list of people with frontend expertise! Some full-stack engineers are not included in @frontend-devs because the primarily do backend development at Sourcegraph, or they aren't very familiar with React, etc.
```

Comment:
```suggestion
Note that this is _not_ intended to be a comprehensive list of people with frontend expertise! Some full-stack engineers are not included in @frontend-devs because the primarily do backend development at Sourcegraph, or they aren't very familiar with React, etc.
```